### ISSUE TITLE
Support for Windows and MacOS

### ISSUE THREAD

**Issue by listepo** [zoontek/react-native-bootsplash#538]
### Why it is needed?

It would be nice to have support for MacOS and Windows

**Comment by HunainHumail** [comment]
Yes, as React Native is evolving and there are alot of applications which are being built in react native windows and react native macos. So we need support for Splash screen there. Right now I have to do it natively which is hectic
